sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -255,24 +246,14 @@ internal class RadarApiClient(
         if (groups?.isNotEmpty() == true) {
             queryParams.append("&groups=${groups.joinToString(separator = ",")}")
         }
-        queryParams.append("&deviceType=${RadarUtils.deviceType}")
```

Comment:
similar q here as the one I had on the iOS PR - seems like we're duplicating params for certain endpoints (track) and not others, intended?